controllers/datafoundationclient.go

```diff
@@ -73,9 +80,9 @@ func (r *dataFoundationClientReconciler) initReconciler(reconciler *ManagedFusio
 	r.storageClient.Name = "ocs-storageclient"
 	r.storageClient.Namespace = offering.Namespace
 
-	r.defaultBlockStorageClassClaim.Name = "ocs-storagecluster-ceph-rbd"
+	r.defaultBlockStorageClassClaim.Name = defaultBlockStorageClassClaimName
 
-	r.defaultFSStorageClassClaim.Name = "ocs-storagecluster-cephfs"
+	r.defaultFSStorageClassClaim.Name = defaultFSStorageClassClaimName
```

Comment:
@ezio-auditore Are we changing the claimNames? It might create issues during migration as on 4.11 we have claims created with ocs-storagecluster-ceph-rbd and ocs-storagecluster-cephfs

Reply:
Not changing the claim names but refactoring it and having the value as a const